client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -310,6 +313,7 @@ class ActivityLogItem extends Component {
 	renderSummary() {
 		const {
 			allowRestore,
```

Comment:
Went with `hideRestore` in 4ec0b82

Inverting to `hideRestore` with prop omission being falsy has some nice properties, simplifies things.